### PR TITLE
Remove Cutlass and meta-buildpack testing

### PR DESCRIPTION
**Pull request by Malax**
`libcnb-test` current doesn't support testing meta buildpacks. We used the Ruby CNB testing library `Cutlass` that predated all of `libcnb-test` in this repo when most of the buildpack were still written in bash.

To reduce complexity, this PR removes Cutlass altogether. This means the meta-buildpacks in this repository are no longer integration tested. I believe this is fine since:

- We only had three (3) tests, all of them were smoke tests.
- Our builder repository already has all the smoke tests we want for each language meta buildpack.
  - Testing there is closer to what the customer will work with. For example, we have partially shimmed buildpacks there that aren't even in the `buildpacks-jvm` repository.
- Testing the combination of JVM buildpacks already happens via `libcnb-test`. For example, the Maven buildpack needs the JVM buildpack and always tests them together.
- The combinations in the meta buildpack are straightforward. There is only one useful way of combining them for each language anyway - so we're not leaving some permutations untested.

In turn, we get a simpler repository and faster CI. I think this is a worthwhile tradeoff. In addition, `libcnb-test` will get support for meta-buildpack testing in the near future, so if we decide we need meta buildpack testing in this repo, we can use native tooling instead of `Cutlass`.

Note: CI currently fails because GitHub is requiring the `Cutlass` suites to pass. After approval of this PR, I will remove the `Cutlass` workflows from the list of workflows that are required for merge.

### CODE REVIEW

File: test/meta-buildpacks/java-function/build.sh [hunk 1]
```diff
@@ -1 +0,0 @@
-../../../common/meta-build.sh
```
Comment (edmorley) [inline test/meta-buildpacks/java-function/build.sh:1]: (Optional) The `common/meta-build.sh` script appears to be unused after this change (though you may be wanting to keep it for adhoc local testing; up to you)

File: test/specs/java-function/function_spec.rb [hunk 2]
```diff
@@ -1,23 +0,0 @@
-require_relative "spec_helper"
-require "digest"
-
-describe "Heroku's Java CNB" do
-  it "generates a callable salesforce function" do
-    Cutlass::App.new("simple-function").transaction do |app|
```
Comment (edmorley) [inline test/specs/java-function/function_spec.rb:6]: I believe the fixture `test-fixtures/simple-function/` is unused after this change

Comment (Malax) [reply test/specs/java-function/function_spec.rb:6]: Correct, thanks! :) Will remove!

Comment (Malax) [reply test/specs/java-function/function_spec.rb:6]: I will keep it for now and add a smoke test to the `java-function` buildpack that will use it. Thanks for pointing this out! 💜

File: test/meta-buildpacks/java-function/build.sh (hunk 1 above)
Comment (Malax) [reply test/meta-buildpacks/java-function/build.sh:1]: Removed in 8a7bc86

File: test/specs/java-function/function_spec.rb (hunk 2 above)
Comment (Malax) [reply test/specs/java-function/function_spec.rb:6]: Test added in [d983283](https://github.com/heroku/buildpacks-jvm/pull/472/commits/d983283192ae2795e3c8cce3bfecb9a5011cda30)

**Pull request merged**